apis/composer/v1alpha1/environment_types.go

```diff
@@ -0,0 +1,536 @@
+	DiskSizeGB *int32 `json:"diskSizeGB,omitempty"`
+
+	// Optional. The set of Google API scopes to be made available on all
+	//  node VMs. If `oauth_scopes` is empty, defaults to
+	//  ["https://www.googleapis.com/auth/cloud-platform"]. Cannot be updated.
+	//
+	//  This field is supported for Cloud Composer environments in versions
+	//  composer-1.*.*-airflow-*.*.*.
+	// +kcc:proto:field=google.cloud.orchestration.airflow.service.v1.NodeConfig.oauth_scopes
+	OauthScopes []string `json:"oauthScopes,omitempty"`
+
+	// Optional. The Google Cloud Platform Service Account to be used by the node
+	//  VMs. If a service account is not specified, the "default" Compute Engine
+	//  service account is used. Cannot be updated.
+	// +kcc:proto:field=google.cloud.orchestration.airflow.service.v1.NodeConfig.service_account
+	ServiceAccountRef *refs.IAMServiceAccountRef `json:"serviceAccountRef,omitempty"`
+
+	// Optional. The list of instance tags applied to all node VMs. Tags are used
+	//  to identify valid sources or targets for network firewalls. Each tag within
+	//  the list must comply with [RFC1035](https://www.ietf.org/rfc/rfc1035.txt).
+	//  Cannot be updated.
+	// +kcc:proto:field=google.cloud.orchestration.airflow.service.v1.NodeConfig.tags
+	Tags []string `json:"tags,omitempty"`
+
+	// Optional. The configuration for controlling how IPs are allocated in the
+	//  GKE cluster.
+	// +kcc:proto:field=google.cloud.orchestration.airflow.service.v1.NodeConfig.ip_allocation_policy
+	IPAllocationPolicy *IPAllocationPolicy `json:"ipAllocationPolicy,omitempty"`
+
+	// Optional. Deploys 'ip-masq-agent' daemon set in the GKE cluster and defines
+	//  nonMasqueradeCIDRs equals to pod IP range so IP masquerading is used for
+	//  all destination addresses, except between pods traffic.
+	//
+	//  See:
+	//  https://cloud.google.com/kubernetes-engine/docs/how-to/ip-masquerade-agent
+	// +kcc:proto:field=google.cloud.orchestration.airflow.service.v1.NodeConfig.enable_ip_masq_agent
+	EnableIPMasqAgent *bool `json:"enableIPMasqAgent,omitempty"`
+
+	// Optional. Network Attachment that Cloud Composer environment is connected
+	//  to, which provides connectivity with a user's VPC network. Takes precedence
+	//  over network and subnetwork settings. If not provided, but network and
+	//  subnetwork are defined during environment, it will be provisioned. If not
+	//  provided and network and subnetwork are also empty, then connectivity to
+	//  user's VPC network is disabled. Network attachment must be provided in
+	//  format
+	//  projects/{project}/regions/{region}/networkAttachments/{networkAttachment}.
+	//
+	//  This field is supported for Cloud Composer environments in versions
+	//  composer-3.*.*-airflow-*.*.* and newer.
+	// +kcc:proto:field=google.cloud.orchestration.airflow.service.v1.NodeConfig.composer_network_attachment
+	ComposerNetworkAttachmentRef *refs.ComputeNetworkAttachmentRef `json:"composerNetworkAttachmentRef,omitempty"`
```

Comment:
cc  @gemmahou looks like more and more resources are referencing the compute resource.